app/views/body/show.html.erb

```diff
@@ -28,7 +28,9 @@
       <%- end -%>
       <%- unless @latest_paper.nil? -%>
         Neueste Anfrage vor <%= relative_time_dativ @latest_paper.published_at %> veröffentlicht
+        <br/>
       <%- end -%>
+      <a href="./<%= @body.friendly_id %>/abo">Neue Anfragen per Mail erhalten!</a>
```

Comment:
`<%= link_to 'Neue Anfragen per E-Mail erhalten', body_subscribe_path(@body) %>` is the rails way of doing this :)